docs/source/GetStartedGuide.rst

```diff
@@ -140,9 +140,6 @@ The output might look like this::
 These Python files are considered by Avocado to contain ``INSTRUMENTED``
 tests.
 
-.. These should refer to proper simple tests example but they are currently
-   lacking in our tree. See GitHub issue #628.
-
```

Comment:
Oh, my fault, I don't have it installed and I thought it ends in the same location as the `example/tests`. Let me double check...